front/pages/w/[wId]/builder/data-sources/public-urls.tsx

```diff
@@ -108,7 +108,11 @@ export const getServerSideProps = withDefaultUserAuthRequirements<{
         },
         "Connector not found"
       );
-      throw new Error("Connector not found");
+      // Not super clean but at least we don't crash the page.
+      return {
```

Comment:
Or we could decide to filter those out so they are not diplayed at all to the user, until we delete them ourselves